docs/assets/js/_partials/chart-recommended-git-versions.js

```diff
@@ -0,0 +1,118 @@
+class RecommendedGitVersionsChart extends AreaChart
+{
+    constructor(canvas, actionBar)
+    {
+        const config =
+        {
+            views:
+            [
+                {
+                    label: '2 m',
+                    tooltip: 'Show the last 2 months',
+                    aggregate: false,
+                    slice: [0, 61],
+                },
+                {
+                    label: '2 y',
+                    tooltip: 'Show the last 2 years',
+                    aggregate:
+                    {
+                        period: 'week',
+                        method: 'first'
+                    },
+                    slice: [0, 106],
+                    default: true,
+                },
+                {
+                    label: 'all',
+                    tooltip: 'Show all data',
+                    aggregate:
+                    {
+                        period: 'week',
+                        method: 'first',
+                    },
+                },
+            ],
+        };
+
+        super(config, canvas, actionBar);
+    }
+
+    render()
+    {
+        const gitVersionsDatabase = new GitVersionsDatabase();
+        const dataURL = '{{ site.dataURL }}/git-versions-new.tsv';
```

Comment:
Should we rename the `git-versions-new.tsv` to git-versions.tsv' in this release? Or maybe `git-versions-v2.tsv` to avoid trouble?